packages/ckeditor5-table/tests/utils/structure.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
+import TableEditing from '../../src/tableediting';
+
+import { setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import { modelTable } from '../_utils/utils';
+import { getHorizontallyOverlappingCells, getVerticallyOverlappingCells } from '../../src/utils/structure';
+
+describe( 'table utils', () => {
```

Comment:
Missing tests for:

*   cropTableToDimensions
*   splitHorizontally
*   splitVertically
*   trimTableCellIfNeeded

Reply:
Not part of the PR.